nekoyume/Assets/_Scripts/UI/Module/Item/ShopItemView.cs

```diff
@@ -75,7 +75,16 @@ public void Set(ShopItem model, Action<ShopItem> onClick)
 
             baseItemView.CountText.gameObject.SetActive(model.ItemBase.ItemType == ItemType.Material);
             baseItemView.CountText.text = model.OrderDigest.ItemCount.ToString();
-            baseItemView.PriceText.text = model.OrderDigest.Price.GetQuantityString();
+
+            if (model.OrderDigest.ItemCount > 1 && decimal.TryParse(model.OrderDigest.Price.GetQuantityString(), out var price))
+            {
+                var priceText = decimal.Ceiling(price / model.OrderDigest.ItemCount * 1000) / 1000;
```

Comment:
Is there a reason for ceiling?
Is there a working that `(price / model.OrderDigest.ItemCount)` alone can't do?

Reply:
> Is there a reason for ceiling?

I have used `Ceiling` to truncate at 4th decimal point, rounded up.

And also, to prevent users from under-calculating the item prices. I think it's better to over-calculate the prices rather then under-calculate them, to get more grasp of the price. 

> Is there a working that `(price / model.OrderDigest.ItemCount)` alone can't do?

This would result in the price text's decimal digits being longer than 3.

There is a way to format them nicely with [`String.Format`](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings#numeric-format-specifier-n), but this just truncates decimal digits. The documenation mentions that _'To perform a rounding operation, use the [Math.Ceiling](https://learn.microsoft.com/en-us/dotnet/api/system.math.ceiling), [Math.Floor](https://learn.microsoft.com/en-us/dotnet/api/system.math.floor), or [Math.Round](https://learn.microsoft.com/en-us/dotnet/api/system.math.round) method.'_.

I am not very exprienced with C#. I will try to find a better method, but this is what I can come up, so far.

Reply:
Thx for your answer and contribution. I understood your intent.
In my humble opinion, 'Decimal.Round' will help you. reference: https://learn.microsoft.com/ko-kr/dotnet/api/system.decimal.round?view=net-6.0#system-decimal-round(system-decimal-system-int32)

example⬇
```suggestion
                var priceText = decimal.Round(price / model.OrderDigest.ItemCount, 3);
```

Reply:
Ah, that would work too. Actually, I read the document you have provided and it seems that `Decimal.Round` also has ceiling capability.

```csharp
var priceText = decimal.Round(price / model.OrderDigest.ItemCount, 3, MidpointRounding.ToPositiveInfinity);
```

But...

> I think it's better to over-calculate the prices rather then under-calculate them, to get more grasp of the price.

I am not sure if it's right to round up the value, rather than rounding up or down depending on the value (rounding, not ceiling). This is just my perspective on this feature.

Reply:
I don't think it'll be a problem.
Even the existing code for calculating the price per unit does not tell you the exact value.

reference: https://github.com/planetarium/NineChronicles/blob/b57b1fd1e078f6020ebfd57eabf191ef78e3c823/nekoyume/Assets/_Scripts/UI/Widget/ShopSell.cs#L472
```cs
var count = model.Count.Value;
var unitPrice = price / count;

var currency = model.UnitPrice.Value.Currency;
var major = (int)unitPrice;
var minor = (int)((Math.Truncate((unitPrice - major) * 100) / 100) * 100);

var fungibleAsset = new FungibleAssetValue(currency, major, minor);
model.UnitPrice.SetValueAndForceNotify(fungibleAsset);
```

Reply:
I also agree with @sonohoshi round is right